documentation/docs/10-getting-started/30-project-structure.md

```diff
@@ -58,7 +50,7 @@ If you added [Vitest](https://vitest.dev) when you set up your project, your uni
 
 ### static
 
-Any static assets that should be served as-is, like `robots.txt` or `favicon.png`, go in here.
+Any static assets that should be served as-is, like `robots.txt` or `favicon.png`, go in here. This directory is optional.
```

Comment:
If we're going to denote this as optional we should probably note that `tests` and `tsconfig.json` can be optional as well

Reply:
Alrighty, I made another pass.